examples/packages/component-library-example/package.json

```diff
@@ -16,37 +16,37 @@
   "license": "MIT",
   "private": true,
   "dependencies": {
-    "@pixeloven-react/common": "6.0.0-beta.32"
+    "@pixeloven-react/common": "6.0.0-beta.33"
   },
   "devDependencies": {
-    "@pixeloven/cli": "6.0.0-beta.32",
-    "@pixeloven-react/testing": "6.0.0-beta.32",
+    "@pixeloven/cli": "6.0.0-beta.33",
+    "@pixeloven-react/testing": "6.0.0-beta.33",
     "lint-staged": "9.5.0"
   },
   "scripts": {
     "compile": "pixeloven compile ts",
     "lint": "yarn lint:ts && yarn lint:scss",
-    "lint:ts": "pixeloven lint ts 'src/**/*.{ts,tsx}'",
-    "lint:scss": "pixeloven lint ts 'src/**/*.{scss}'",
+    "lint:ts": "pixeloven lint ts ./src/**/*.{ts,tsx}",
+    "lint:scss": "pixeloven lint ts ./src/**/*.{scss}",
     "postcompile": "pixeloven copy assets",
     "precommit": "lint-staged",
     "pretest": "pixeloven delete coverage",
     "pretty": "yarn run pretty:ts && yarn run pretty:scss",
-    "pretty:ts": "pixeloven pretty ts 'src/**/*.{ts,tsx}'",
-    "pretty:scss": "pixeloven pretty scss 'src/**/*.scss'",
+    "pretty:ts": "pixeloven pretty ./src/**/*.{ts,tsx}",
```

Comment:
Should this one keep the `ts` argument?

Reply:
Looking at the other examples above it looks like maybe the `scss` argument below should be removed

Reply:
Updated